l10n_br_fiscal/models/tax_definition.py

```diff
@@ -21,7 +20,10 @@ class TaxDefinition(models.Model):
     _description = "Tax Definition"
 
     def _get_complete_name(self):
-        return f"{self.tax_group_id.name}-{self.tax_id.name}-{self.cst_code}"
+        if self.is_benefit:
+            return f"{self.tax_group_id.name}-BENEF-{self.tax_id.name}-{self.cst_code}"
```

Comment:
se for um beneficio não vai ter o tax_id.name nem o cst_code, ai pode deixar de fora, se não vai imprimir FLASE-FALSE